_data/footer.yml

```diff
@@ -50,10 +50,10 @@ contact:
       href: https://facebook.com/NSAUSGov/
       external: true
       type: facebook
-    - text: Twitter
-      href: https://twitter.com/nsagov
+    - text: X
+      href: https://x.com/nsagov
       external: true
-      type: twitter
+      type: x
```

Comment:
The 'type' field must be supported by the jekyll remote theme. We're using https://github.com/18F/uswds-jekyll version 4.2.0, which does not support `type: x` yet. It would require extra effort to migrate to one of the newer versions of the themes. We could change the text and the link, but not the type (which controls the image shown). However, given that it currently redirects to X, I don't see much point in updating the footer at all, until we migrate to the new theme.

Reply:
Fair enough should I just close this?

Reply:
I see sorry

Reply:
@nonetrix Hey, no worries. Thanks for making an attempt at updating this. Even though we can't use the change right now, I appreciate your willingness to help out!